grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -25,6 +25,93 @@ import spock.lang.Unroll
 class GrabbitConfigurationSpec extends Specification {
 
     @Unroll
+    def "Should return exclude paths"() {
+        when:
+        def output = GrabbitConfiguration.create(input)
+
+        then:
+        output instanceof GrabbitConfiguration
```

Comment:
I don't understand what this is testing. What other possible state could this be in?